src/qt/nametablemodel.cpp

```diff
@@ -98,7 +103,156 @@ class NameTablePriv
                 std::string data = maybeData.get_str();
                 int height = find_value ( v, "height").get_int();
                 int expiresIn = find_value ( v, "expires_in").get_int();
-                vNamesO[name] = NameTableEntry(name, data, height, expiresIn, "confirmed");
+
+                bool isMine = find_value ( v, "ismine").get_bool();
+                bool isExpired = find_value ( v, "expired").get_bool();
```

Comment:
Nit: Please make them `const`.  Also the whitespace is again weird, but I see it matches the lines above.